moveit_ros/visualization/rviz_plugin_render_tools/src/trajectory_visualization.cpp

```diff
@@ -399,13 +399,17 @@ void TrajectoryVisualization::update(float wall_dt, float ros_dt)
     if (animating_path_)
     {
       current_state_ = -1;
-      current_state_time_ = std::numeric_limits<float>::infinity();
```

Comment:
I think, setting to infinity here, is needed to trigger the display of the first way point, isn't it?

Reply:
This is indeed the case, however I can no longer do it this way because current_state_time_ isn't reset to zero after each new state display. To address this, I can change the comparison in line 419 from > to >=.

Reply:
I just noticed that the line below forces the display of the first way point. Nevertheless I think this would be a good change.